json/entities/entities.schema.json

```diff
@@ -7,122 +7,111 @@
-            "type": "string",
-            "description":"In case of a person mention, the entity component of type name."
+            "description": "The surface form of the named entity mention, as it appears in the text."
           },
           "lOffset": {
             "type": "integer",
-            "description":"The left character offset of the named entity with respect to the content item, as in the rebuilt format."
+            "description": "The left character offset of the named entity with respect to the content item."
           },
           "rOffset": {
             "type": "integer",
-            "description":"The right character offset of the named entity with respect to the content item, as in the rebuilt format."
-          },          
-          "firstname": {
-            "type": "string",
-             "description":"In case of a person mention and if available, the first name."
+            "description": "The right character offset of the named entity with respect to the content item."
           },
-          "surname": {
-            "type": "string",
-             "description":"In case of a person mention and if available, the surname."
+          "nested": {
+            "type": "boolean",
+            "description": "Indicates whether the mention is nested."
           },
-          "title": {
-            "type": "string",
-            "description":"In case of a person mention, the entity component of type 'title'."
+          "confidence_ner": {
+            "type": "number",
+            "description": "Confidence score of the Named Entity Recognition process."
           },
-          "function": {
+          "confidence_nel": {
+            "type": "number",
+            "description": "Confidence score of the Named Entity Linking process."
+          },
+          "wkd_id": {
             "type": "string",
-            "description":"In case of a person mention, the entity component of type 'function'."
+            "description": "Wikidata QID if available."
           },
-          "demonym": {
+          "wkpedia_pagename": {
             "type": "string",
-            "description":"In case of a person mention, the entity component of type 'demonym'."
+            "description": "Wikipedia page name or URL in the relevant language."
           },
-          "nested": {
-            "type": "boolean",
-            "description":"In case of a nested mention, this property should be set to true. Can be ignored if not."
+          "name": {
+            "type": "string",
+            "description": "Full name of the entity, including first, middle, and last names as well as initials and nicknames, if applicable."
           },
-          "wkd_id": {
+          "title": {
             "type": "string",
-            "description":"If exists, wikidata QID"
+            "description": "Title or designator of a person (e.g., 'Herr Chirac', 'Son Altesse le prince Rainier'), if applicable."
```

Comment:
I see a bit the risk of (re)defining guidelines here, and it should be mentioned that this is only for a certain entity type (even though it's obvious). I would suggest to keep it more neutral as in the original, adding however a reference to the impresso hipe guidelines for further information, e.g.:

```
 "title": {
            "type": "string",
            "description":"In case of a person mention, the entity component of type 'title'., as defined in the Impresso HIPE NE Annotation guidelines (https://zenodo.org/records/3585750). "
          },
```